README.rst

```diff
@@ -51,12 +51,23 @@ The following configuration values are available:
   Other typical values includes ``PCM``. Run the command ``amixer scontrols``
   to list available controls on your system.
 
+- ``alsamixer/min_volume`` and ``alsamixer/max_volume``: Map the Mopidy volume control range to
+  a different range. Values are in the range 0-100. Use this if the default range (0 - 100)
+  is too wide, resulting in a small usable range for Mopidy's volume control.
+  For example try ``min_volume = 30`` and ``max_volume = 70`` to map Mopidy's volume control to the middle
+  of ALSA's volume range.
+
+- ``alsamixer/volume_scale``: Either 'linear', 'cubic' or 'db'. The cubic scale is recommended as it produces
+  better results to the ear, and matches the volume scale in alsamixer. The default is the linear scale.
```

Comment:
It's a bit funny that we say cubic is better, but we use linear anyway... Should we just make cubic the default? Personally I don't use this mixer so I probably shouldn't decide this, but doesn't hurt to ask :-)